e2e/cypress/integration/channel/message_reply_spec.js

```diff
@@ -49,15 +48,9 @@ describe('Message Reply', () => {
         cy.postMessage('Two');
 
         cy.get('@yesterdaysPost').then((postId) => {
-            // # Scroll up to yesterdays message
-            cy.get(`#post_${postId}`).scrollIntoView();
-
             // # Open RHS comment menu
             cy.clickPostCommentIcon(postId);
 
-            // # Upload an attachment to the reply
-            cy.fileUpload('#fileUploadInput').wait(TIMEOUTS.TINY);
```

Comment:
This step is actually not relevant to main test.  I'll investigate separately why this one is causing some kind of error - `scrollIntoView`

Reply:
Actually, we have separate ticket for this particular case - https://mattermost.atlassian.net/browse/MM-16731